### PR TITLE
Docs/develop

### PR DESCRIPTION
**Pull request by vincent-inviseo**
complete docs.develop.rst with override route and complete plugin_install.rst with how uninstall plugin

### CODE REVIEW

File: docs/plugin_install.rst [hunk 1]
```diff
@@ -49,3 +49,13 @@ List PyScada plugin installed
     # activate the PyScada virtual environment
     source /home/pyscada/.venv/bin/activate
     pip3 list | grep cada
+
+
+Uninstall plugin
```
Comment (clavay) [inline docs/plugin_install.rst:54]:
```suggestion
Uninstall a plugin
```

File: docs/develop.rst [hunk 2]
```diff
@@ -78,3 +78,39 @@ Otherwise restart PyScada.
...
+
+* python virtual environment installation: located in ``/var/www/pyscada/PyScadaServer/PyScadaServer``
+* Docker installation: located in ``/src/pyscada/PyScadaServer/PyScadaServer``
+
+
+By default, the project's ``urls.py`` file loads only the ``urls.py`` file from ``pyscada.core``. The ``pyscada.core.urls`` file loads all the other modules ``urls.py`` files in random order.
+
+The route used is the first valid one encountered, so if you want to replace an existing route, you have to load your route before the others, i.e. before loading ``pyscada.core.urls`` file.
+
+To do this, you need to modify your project's urls.py file.
```
Comment (clavay) [inline docs/develop.rst:97]:
```suggestion
To do this, you need to modify your project's ``urls.py`` file.
```

File: docs/develop.rst [hunk 3]
```diff
@@ -78,3 +78,39 @@ Otherwise restart PyScada.
 .. code-block:: shell
 
     sudo systemctl restart pyscada
+
+
+Override routes
+----------------
+
+This use case is encountered when you wish to rewrite an existing view (and therefore an existing route).
```
Comment (clavay) [inline docs/develop.rst:86]:
```suggestion
This use case is encountered when you wish to rewrite an existing view (and therefore an existing route).

```

File: docs/plugin_install.rst [hunk 4]
```diff
@@ -49,3 +49,13 @@ List PyScada plugin installed
...
+----------------------
+
+To uninstall a plugin
+
+.. code-block:: shell
+    
+    sudo -u pyscada -E env PATH=${PATH} pip uninstall 'yourPlugin'
```
Comment (clavay) [inline docs/plugin_install.rst:61]:
```suggestion
    sudo -u pyscada -E env PATH=${PATH} pip uninstall yourPlugin
```

**Review by clavay — CHANGES_REQUESTED** [review]
Please update as proposed.

@trombastic are you ok ?

File: docs/develop.rst (hunk 3 above)
Comment (vincent-inviseo) [reply docs/develop.rst:86]: Hi, @clavay i don't see modification to do between my file and here

Comment (clavay) [reply docs/develop.rst:86]: There is a new line (87).

**Pull request closed without merging**